lib/runtime/addon.js

```diff
@@ -341,4 +340,34 @@ export default class Addon {
     return configModule && (configModule.default || configModule);
   }
 
+  get blueprintNames() {
+    let names = fs.readdirSync(path.join(this.dir, 'blueprints'));
+    return names;
```

Comment:
Actually, on second thought - what if we got rid of this method entirely. If you want the names of the blueprints, it's just `addon.blueprints.map((blueprint) => blueprint.name)`? Reduces the API surface area of the Addon class, and reduces the Addon class coupling to the Blueprint implementation.